engine/src/main/scala/cromwell/engine/workflow/lifecycle/WorkflowExecutionActor.scala

```diff
@@ -221,6 +233,19 @@ final case class WorkflowExecutionActor(workflowId: WorkflowId, workflowDescript
       log.info(s"$tag transitioning from $fromState to $toState.")
   }
 
+  private def handleRetryableFailure(jobKey: BackendJobDescriptorKey) = {
```

Comment:
Will do that. 
Perhaps, do you think it's fair to overwrite the old entry in the execution store in this way?